client/package.json

```diff
@@ -28,6 +28,7 @@
     "build:ui": "run-s build:css build:js",
     "build:asset-deps": "mkdir -p public/assets/rcue && cp -R node_modules/rcue/dist/* public/assets/rcue",
     "build:asset-images": "mkdir -p public/assets/images && cp -R src/styles/images/* public/assets/images",
+    "build:asset-css": "npm run build:css && mkdir -p public/assets/css && cp -R src/styles/css/* public/assets/css",
```

Comment:
This does cause the css to get built twice, once or the the build:ui and once here for build:assets-css. Not sure the best way around this since build:js does the copy from build to public so the public/asses/css must be done before that. build:ui should build all the local ui stuff (including css) but build:assets should build the complete set of assets.

Reply:
Gotcha. Had been looking at integrating the CSS/SCSS/LESS into the Create React App setup (using imports on JS files). That would definitely affect this, possibly making it so the ```build:ui``` has to run since CSS and JS would be integrated. 

I think what you have is good for this round, and it gets us thinking about what needs to take place, next round. 

From what I can gather all we really need is a link to the CSS and possibly images from the Django side. We could keep it simple and manually link those resources once the build gets checked in. Or the slightly more complicated opposite end approach... and we could look at integrating the Django templates into our side under the public directory. So instead those get copied over. They wouldn't necessarily run on our side, but we could use the Create React App to fill in the paths for us... if it writes them out as strings.